src/mds/objects.py

```diff
@@ -89,7 +89,7 @@ async def create_object(
     try:
         # NOTE: token can be None if no Authorization header was provided, we expect
         #       this to cause a downstream exception since it is invalid
-        token_claims = await access_token("user", "openid", purpose="access")(token)
+        token_claims = await access_token("user", "openid", issuer=config.ALLOWED_ISSUERS[0] if config.FORCE_ISSUER else None, allowed_issuers=config.ALLOWED_ISSUERS if config.FORCE_ISSUER else None, purpose="access")(token)
```

Comment:
Make sure to run Black formatting against this, it's a very long line. We tend to not embed conditional logic into parameters for a function call (though this is more in practice rather than enforced). I'd _prefer_ to see logic before this call like this. It's more code, but imo it's more readable. Ultimately I guess this is up to you

```
issuer = None
allowed_issuers = None

# override token iss 
if config.FORCE_ISSUER:
  issuer = config.ALLOWED_ISSUERS[0]
  allowed_issuers =  config.ALLOWED_ISSUERS
  
 token_claims = await access_token("user", "openid", issuer=issuer, allowed_issuers=allowed_issuers, purpose="access")(token)
```